core/src/util/vboMesh.cpp

```diff
@@ -73,10 +71,10 @@ void VboMesh::subDataUpload() {
     if (m_dirtySize != 0) {
         glBindBuffer(GL_ARRAY_BUFFER, m_glVertexBuffer);
 
-        long vertexBytes = m_nVertices * m_vertexLayout->getStride();
+        size_t vertexBytes = m_nVertices * m_vertexLayout->getStride();
 
         // updating the entire buffer
-        if (std::abs(m_dirtySize - vertexBytes) < m_vertexLayout->getStride()) {
+        if (vertexBytes - m_dirtySize < m_vertexLayout->getStride()) {
 
```

Comment:
Ah, I made this edit along with some of the #include changes, could have been better documented. I only changed it to prevent a warning about signed/unsigned comparison. I believe this comparison could also reasonably be "<= 0" but I was only concerned with the warning.